daikon-logging/logging-event-layout/src/main/resources/mdc_ecs_mapping.properties

```
@@ -1,4 +1,4 @@
-accountId=cloud.account.id
+accountId=labels.customer_account_id
```

As discussed, `organization.id` seems to be the more suitable field for storing account/tenant id.
```suggestion
accountId=organization.id
```